client/src/deep-dive/planning-for-browser-support.tsx

```diff
@@ -0,0 +1,456 @@
+import { useParams } from "react-router-dom";
+
+import ArticleMeta from "../ui/molecules/article-meta";
+import Byline from "../ui/organisms/byline";
+import DeepDivesFeature from "../ui/organisms/deep-dives-feature";
+import SeriesCard from "../ui/molecules/series-card";
+import Survey from "../ui/organisms/survey";
+
+import "./index.scss";
+
+export default function PlanningForBrowserSupport() {
+  const { locale } = useParams();
+  const seriesList = [
+    {
+      displayName: "Planning for browser support",
+      url: "planning-for-browser-support",
+      state: "active",
+    },
+    {
+      displayName: "Your browser support toolkit",
+      url: "your-browser-support-toolkit",
+    },
+    {
+      displayName: "Practical browser support",
+      url: "#",
+      state: "unavailable",
+    },
+  ];
+
+  return (
+    <>
+      <div className="main-article-page-content-container girdle">
+        <article className="deep-dive-article-container">
+          <header className="main-heading-group heading-group">
+            <h1>Planning for browser support</h1>
+            <h2>Modern CSS in the real world : Part one</h2>
+          </header>
+          <p className="article-lead">
+            Learn about the types of CSS issues you’ll be facing when working
+            with your team in a web-based environment and tips for developing a
+            future-proof support strategy.
+          </p>
+          <Byline
+            avatar="rachel-andrew.png"
+            author="Rachel Andrew"
+            authorDescription="MDN Editor &amp; Writer, Former Editor in Chief of Smashing Magazine"
```

Comment:
```suggestion
            authorDescription="MDN Editor & Writer, Former Editor in Chief of Smashing Magazine"
```
when that gets rendered by the `<Byline>` component, it will escape.